qstack/spahm/rho/utils.py

```diff
@@ -31,13 +31,22 @@ def get_chsp(fname, n):
 
 def load_mols(xyzlist, charge, spin, basis, printlevel=0, units='ANG', ecp=None):
     mols = []
+    if printlevel > 0:
+        progress = add_progressbar(max_value=len(xyzlist))
+        i = 0
     for xyzfile, ch, sp in zip(xyzlist, charge, spin):
-        if printlevel>0: print(xyzfile, flush=True)
+        if printlevel>1: print(xyzfile, flush=True)
```

Comment:
don't know what's best to handle different output printing levels ?
- I'm imagining one level that would trigger the `progressbar` and would be set by default in the scripts you can run from CLI
- but also a lower one when calling the function in your own scripts (without `progressbar`)
    *to avoid mixing `print()` with the progressbar*

Reply:
we could pass a tuple `(printlevel, print_progress)`

Reply:
but should we use progressbar2 or tqdm?

Reply:
this is don't know!

I tried a few things with the `progressbar2` to make it work when `sbatch`-ing to SLURM, but unsuccessfully (the progressbar does not get overwritten/replace but rather extended on a new line). Same behaviour when mixed with more output printings.

_do you know if `tqdm` can better handle these situations?_

Reply:
moved to `tqdm` (ebb3f5b)